terraform/modules/parameters/shared-attributes.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_ssm_parameter" "shared_attributes_signing_cert" {
+  name        = "/${var.environment}/ipv/core/sharedAttributes/signingCert"
+  description = "The IPV core's shared attributes signing certificate"
```

Comment:
We could add something like `This is used by CRIs to validate the shared attributes JWT signature`